src/App.jsx

```diff
@@ -2,13 +2,28 @@ import "./App.css";
 import Footer from "./pages/Footer";
 import Header from "./pages/Header";
 import Home from "./pages/Home";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+
+
 
 function App() {
+
+  const appRouter = createBrowserRouter([
+    {
+        path : "/",
+        element : <Home/>
+    },
+    {
+        path : "Header",
```

Comment:
link paths should never in capital and try to separate the routing logic to the file like AppLayout not App,js